CRM/Civicase/Hook/BuildForm/TokenTree.php

```diff
@@ -195,18 +195,16 @@ private function setAllRelevantCustomFields() {
    *   Restructured token tree.
    */
   private function addCaseRoleTokens(array $caseRoleTokens, array &$newTokenTree) {
-    $contactRoleCount = 0;
     $contactRoleTokens = [];
     foreach ($caseRoleTokens as $key => $token) {
       if ($token['id'] === '{case_roles.client}') {
         continue;
       }
       $roleName = explode('-', $token['text']);
       $roleName = $roleName[0];
-      $tokenName = 'Case Role ' . $contactRoleCount . ' "' . trim($roleName) . '"';
+      $tokenName = trim($roleName);
       if (empty($contactRoleTokens[$tokenName])) {
-        $contactRoleCount++;
-        $tokenName = 'Case Role ' . $contactRoleCount . ' "' . trim($roleName) . '"';
+        $tokenName = trim($roleName);
```

Comment:
@shahrukh-compucorp this is not needed anymore (it is the same than before the if)